cypress/e2e/gui/detalheCliente.cy.js

```diff
@@ -0,0 +1,22 @@
+/// <reference types="cypress" />
```

Comment:
```suggestion
/// <reference types="cypress" />

```

Consistência nos testes é important.

Visto que no teste anterior há uma linha em branco entre este comentário e o `describe`, melhor seguir a mesma convenção em todos os testes.

Reply:
Obrigada pela dica!